data/json/items/melee.json

```diff
@@ -4066,5 +4066,73 @@
         "bashing": 1,
         "cutting": 0,
         "to_hit": -2
-    }
+    },{
+    "type": "GENERIC",
+    "id": "sawblade_axe",
+    "name": "sawblade axe",
+    "description": "This is an axe made by attaching the blade of a circular saw to a pipe. It makes a decent melee weapon.",
+    "weight": 1600,
+    "to_hit": 1,
```

Comment:
The non-improvised axes are both longer and have better grips than just a pipe. The sawblade axe is more along the lines of a primitive stone or copper axe, which have 0 to_hit but should probably be -1 or even -2.